lib/web/scripts/desktop/configure-ad.ps1

```diff
@@ -1,5 +1,28 @@
 $ErrorActionPreference = "Stop"
 
+# Test that we can create and remove the required files 
+# If the directory prevents writing and removing files the script can fail.
+Try
+{
+ $file_array = @('teleport.pem','windows.der','windows.pem')
+ for($i = 0; $i -lt $file_array.length; $i++){ 
+ Write-Output ""| Out-File -FilePath $file_array[$i]
```

Comment:
I feel like there's a cleaner way to fix this.

Can we remove these checks and use `New-TemporaryFile` to write the cert file later on rather than assuming it's safe to write to the current directory?

Reply:
I will take a look @zmb3

Reply:
@zmb3 `New-TemporaryFile` is not supported in the default powershell in Windows 2012R2.  I have updated it to use the temp dir for the location of the 3 files we use.